src/frontend/dummy_data/posts.json

```diff
@@ -0,0 +1,36 @@
+      "updated": "October 15, 2019",
```

Comment:
Can you make these proper JSON style date strings?

Reply:
Good idea, I've now used the Date object methods to turn a properly formatted string while using the correct JS formatted Timestamp to create the Date object.